slack_bolt/app/app.py

```diff
@@ -373,10 +424,11 @@ def event(
         matchers: Optional[List[Callable[..., bool]]] = None,
         middleware: Optional[List[Union[Callable, Middleware]]] = None,
     ):
-        def __call__(func):
+        def __call__(*args, **kwargs):
```

Comment:
This method accepts either of args: `[ack_func, lazy_func_1, lazy_func_2]`  or kwargs `{"ack": ack_func, "lazy": [lazy_func_1, lazy_func_2]}`